backends/tfhe-cuda-backend/cuda/tests_and_benchmarks/tests/CMakeLists.txt

```diff
@@ -1,6 +1,9 @@
 project(test_tfhe_cuda_backend LANGUAGES CXX)
 enable_language(CUDA)
 
+find_package(CUDA REQUIRED)
+find_package(CUDAToolkit REQUIRED)
+
```

Comment:
Here it should be:
```
roject(tfhe_cuda_backend LANGUAGES CXX)

# See if the minimum CUDA version is available. If not, only enable documentation building.
set(MINIMUM_SUPPORTED_CUDA_VERSION 10.0)
include(CheckLanguage)
# See if CUDA is available
check_language(CUDA)
# If so, enable CUDA to check the version.
if(CMAKE_CUDA_COMPILER)
  enable_language(CUDA)
endif()
# If CUDA is not available, or the minimum version is too low do not build
if(NOT CMAKE_CUDA_COMPILER)
  message(FATAL_ERROR "Cuda compiler not found.")
endif()
```
like in the main CMakelists.txt. I think using `find_package` was resulting in warnings.